lighthouse-core/lib/i18n/i18n.js

```diff
@@ -94,6 +94,20 @@ const UIStrings = {
   otherResourceType: 'Other',
   /** Label for a row in a data table; entries will be the total number and byte size of all third-party resources loaded by a web page. 'Third-party resources are items loaded from URLs that aren't controlled by the owner of the web page. */
   thirdPartyResourceType: 'Third-party',
+  /** The name of the metric that marks the time at which the first text or image is painted by the browser. Shown to users as the label for the numeric metric value. Ideally fits within a ~40 character limit. */
+  firstContentfulPaint: 'First Contentful Paint',
```

Comment:
should we suffix these with `Title`? just to disambiguate with description

Reply:
I thought the `Title` suffix might get confusing when they're used for timing budgets (they won't be used as titles then), so I added the `Metric` suffix instead. Which I think is an improvement? Let me know your thoughts.